de_web_plugin.cpp

```diff
@@ -5002,7 +5002,10 @@ void DeRestPluginPrivate::addSensorNode(const deCONZ::Node *node, const SensorFi
         {
             clusterId = METERING_CLUSTER_ID;
             item = sensorNode.addItem(DataTypeUInt64, RStateConsumption);
-            if ((modelId != QLatin1String("SP 120")) && (modelId != QLatin1String("ZB-ONOFFPlug-D0005")))
+            if ((modelId != QLatin1String("SP 120")) &&
+                (modelId != QLatin1String("ZB-ONOFFPlug-D0005")) &&
+                (modelId != QLatin1String("TS0121")) &&
+                (modelId != QLatin1String("Plug-230V-ZB3.0")))
```

Comment:
Make sure to make these changes to `database.cpp` as well.
https://github.com/dresden-elektronik/deconz-rest-plugin/blob/b3086c6009154aa1b5cdb89febb5f177912952e9/database.cpp#L3162

Reply:
The are, forgot it here ;)